src/data/api-lapis.ts

```diff
@@ -69,16 +80,23 @@ export function getCurrentLapisDataVersionDate(): Date | undefined {
 }
 
 export async function fetchNextcladeDatasetInfo(signal?: AbortSignal): Promise<NextcladeDatasetInfo> {
-  let url = '/info/nextclade-dataset';
-  const res = await get(url, signal, true);
-  if (!res.ok) {
+  let url = '/details?limit=1';
```

Comment:
We have to use the aggregated endpoint to obtain the nextclade data version as the website may only access aggregated data.